src/client/datascience/jupyter/debuggerVariables.ts

```diff
@@ -134,10 +138,8 @@ export class DebuggerVariables extends DebugLocationTracker
             (targetVariable as any).frameId
         );
 
-        let fileName;
-        if (notebook) {
-            fileName = path.basename(notebook.identity.path);
-        } else if (this.debugLocation?.fileName) {
+        let fileName = notebook ? path.basename(notebook.identity.path) : '';
+        if (!fileName && this.debugLocation?.fileName) {
```

Comment:
Found a problem, when debugging in the interactive window `notebook` is undefined, hence should use debugLocation.fileName to populate breadcrumbs